### ISSUE TITLE
Does not work with Rails 5.1

### ISSUE THREAD

**Issue by GuskiS** [customink/secondbase#44]
Not sure why, but doesn't work with Rails 5.1.
```
ActiveRecord::StatementInvalid (PG::UndefinedTable: ERROR:  relation "my_table" does not exist ... )
```
I tries to use `PG` but it should be mysql.

**Comment by cw-octanner** [comment]
I'm seeing this as well.  Even doing a .connect query on the SecondBase::Base class runs on the first db and not the second.  Rollback to Rails 5.0.2 works fine again.

**Comment by metaskills** [comment]
I'll find some time to play with this soon. One thing I noticed today. First PR never made it into Rails core:

* https://github.com/rails/rails/pull/27611
* https://github.com/rails/rails/pull/28095

**Comment by aronwolf90** [comment]
not sure, but maybe it fix it https://github.com/customink/secondbase/pull/48

**Comment by metaskills** [comment]
Published v2.1.4 with Rails v5.1 fixes. Thanks!
